src/components/omniSidebarNav/submenus/terminus.js

```diff
@@ -19,8 +19,10 @@ const terminus = () => {
           '/terminus/supported-terminus',
           'Supported Terminus and PHP Versions',
         ),
-        simpleLink('/terminus/updates', 'Terminus Changelog'),
-        simpleLink('/terminus/terminus-3-0', 'Terminus 3'),
+        simpleLink('/terminus/updates', 'Terminus Changelog', [
+          simpleLink('/terminus/terminus-3-0', 'Terminus 3'),
+          simpleLink('/terminus/terminus-4-0', 'Terminus 4'),
```

Comment:
I think these should be swapped. 
```suggestion
          simpleLink('/terminus/terminus-4-0', 'Terminus 4'),
          simpleLink('/terminus/terminus-3-0', 'Terminus 3'),
```